generators/php/model/src/generateModels.ts

```diff
@@ -1,25 +1,27 @@
-import { RelativeFilePath } from "@fern-api/fs-utils";
 import { PhpFile } from "@fern-api/php-codegen";
+import { EnumTypeDeclaration } from "@fern-fern/ir-sdk/api";
+import { EnumGenerator } from "./enum/EnumGenerator";
 import { ModelGeneratorContext } from "./ModelGeneratorContext";
-import { php } from "@fern-api/php-codegen";
+import { ObjectGenerator } from "./object/ObjectGenerator";
 
 export function generateModels({ context }: { context: ModelGeneratorContext }): PhpFile[] {
     const files: PhpFile[] = [];
-    const clazz = php.class_({
-        name: "Placeholder",
-        namespace: `${context.getRootNamespace()}\\Placeholder`
-    });
-    clazz.addConstructor({
-        body: php.codeblock("// TODO: Implement me!"),
-        parameters: []
-    });
-    files.push(
-        new PhpFile({
-            clazz,
-            rootNamespace: context.getRootNamespace(),
-            directory: RelativeFilePath.of("Placeholder"),
-            customConfig: context.customConfig
-        })
-    );
+    for (const [typeId, typeDeclaration] of Object.entries(context.ir.types)) {
+        const file = typeDeclaration.shape._visit<PhpFile | undefined>({
+            alias: () => undefined,
+            enum: (etd: EnumTypeDeclaration) => {
+                return new EnumGenerator(context, typeDeclaration, etd).generate();
+            },
+            object: (otd) => {
+                return new ObjectGenerator(context, typeDeclaration, otd).generate();
+            },
```

Comment:
nit: Let's spell out the variable name since we do that elsewhere:
```suggestion
            enum: (enumDeclaration: EnumTypeDeclaration) => {
                return new EnumGenerator(context, typeDeclaration, enumDeclaration).generate();
            },
            object: (objectDeclaration) => {
                return new ObjectGenerator(context, typeDeclaration, objectDeclaration).generate();
            },
```